mani_skill/vector/wrappers/gymnasium.py

```diff
@@ -73,7 +74,7 @@ def __init__(
 
     @property
     def device(self):
-        return self.base_env.device
+        return self.base_env.unwrapped.device
```

Comment:
this shouldn't be necessary?

Reply:
I get the following warning when testing : 
```
UserWarning: WARN: env.device to get variables from other wrappers is deprecated and will be removed in v1.0, to get this variable you can do `env.unwrapped.device` for environment variables.
```

I will revert the change still so this PR stays focused.